SCClassLibrary/Common/Audio/bela/BELAUGens.sc

```diff
@@ -97,14 +97,24 @@ DigitalIO : UGen {
     }
 }
 
-/* input 1: bus
- * input 2: number of channels to scope
+
+BelaScopeUGen : UGen {
+    *ar {
+       FreeSelfWhenDone.kr(super.new1('audio'));
+       ^0.0
+    }
+}
+
+/* input 1: channel offset
+ * input 2: array of signals to scope
  */
-BelaScopeUGen : AbstractOut {
-    *ar { arg busnum, numChannels;
-       super.performList('new1', 'audio', In.ar(busnum, numChannels));
-       ^0.0;    // BelaScopeUGen has no outputs
+BelaScopeOut : AbstractOut {
+    *ar {
+		arg offset = 0, channelsArray;
+		channelsArray = this.replaceZeroesWithSilence(channelsArray.asUGenInput(this).asArray);
+		this.multiNewList(['audio', offset] ++ channelsArray)
+		^0.0
     }
-    *numFixedArgs { ^0 }
-    writesToBus { ^false }
+	*numFixedArgs { ^1 }
```

Comment:
whitespace inconsistent with the surrounding lines (it seems)

Reply:
I'll fix them all at once in the end, sorry